handbook/engineering/campaigns/goals.md

```diff
@@ -1,66 +1,69 @@
-# Campaigns goals and priorities
+# Campaigns goals
 
-## Goals
+The campaigns team is building the best solution available for creating and managing large scale changes. To do so we will focus on the following objectives:
 
-### Build a base of 10 customers who have created 80 campaigns and made 4000 changesets in Q4
+## Grow adoption of campaigns 
 
-We want to get a solid set of customers who regularly use and rely on campaigns. 
+We need to grow usage of campaigns to understand our customer use cases and inform our roadmap.
 
-What will get us there?
+How do we do this:
+  * Proactive customer outreach
+  * Continual user validation
+  * Strive to create a frictionless initial experience
+  * Provide amazing support
 
-- **Proactive outreach** Campaigns is new and transformative software. Many companies are not yet aware they need large scale code modifications. We need to find those companies within our customer base and introduce them to campaigns. This process will also teach us about marketing campaigns.
-- **Smoother on-ramp:** Internal testing has revealed that using campaigns for the first time is not as easy as it could be. A frictionless initial experience is critical to adoption and a delightful user experience.
-- **Better debugging experience:** It can be challenging to debug a more complex campaign. Figuring out what went wrong and how to fix it is a slow iterative loop.
-- **User validation:** The above issues were revealed by user testing. Once we believe we have fixed them, we need to validate this with further user testing.
+Our success metrics:
+  * Build a base of 10 customers who have created 80 campaigns and made 4000 changesets in Q4
+  * [Looker dashboard with usage metrics](https://sourcegraph.looker.com/dashboards/136) (internal only)
 
-### Grow adoption of campaigns
 
-Once we’re out of beta, we want everyone to start using campaigns!
+## Design and build the most robust, yet easy to use solution for creating changesets across many repositories
 
-We need to follow-up with customers who are already using campaigns to make sure that we’re taking their input into consideration when prioritizing new features.
+Campaigns need to be easy to explain, easy to understand, easy to start using, and easy to explore. At the same time, it should be the most powerful tool available for making programatic changes to multiple repositories. 
 
-We need to make it as easy as possible for new customers to try out campaigns:
+How do we do this:
+  * Clean conceptual model
+  * Expressive syntax
+  * Understand our customers environments and requirements
+  * Strong alignment of features to developer objectives
+  * Excellent docs, tutorials, demos/training
+  * Best debugging experience
```

Comment:
This feels like a general algorithm rather than a plan. Once you have a clear problem statement, can you turn this list into an ordered set of milestones. Each milestone should wholly or partially address something in the problem statement.